dex/src/main/scala/com/wavesplatform/dex/api/MatcherApiRoute.scala

```diff
@@ -141,12 +141,19 @@ case class MatcherApiRoute(assetPairBuilder: AssetPairBuilder,
         }
     }
 
+  @inline private def askAddressActor(sender: Address, msg: AddressActor.Command): Future[ToResponseMarshallable] =
+    askMapAddressActor[MatcherResponse](sender, msg)(x => x)
```

Comment:
Use `identity` function

Reply:
`identity` doesn't work here because of its signature:
```
def identity[A](x: A): A
```
But here we need a function from `MatcherResponse` to `ToResponseMarshallable`. There is an implicit from `MatcherResponse` to `ToResponseMarshallable`, so `x => x` works perfectly.

Reply:
Got it